projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.ts

```diff
@@ -63,6 +63,7 @@ export class QuoteReviewComponent implements OnInit, OnDestroy {
     this.activeCategory$ = this.categoryService.getActiveCategory();
     this.baseUrl = this.config.backend.occ.baseUrl || '';
     this.displayQuoteStatusPendingMessage();
+    this.setBindingState();
   }
```

Comment:
Changed the place of setting local storage to prevent losing data if user change browser or device.